test/inspector/test-on-off.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testResume(session) {
+  session.sendCommandsAndExpectClose([
+    { 'method': 'Runtime.runIfWaitingForDebugger' }
+  ]).expectStderrOutput('Debugger listening on');
+}
+
+function runTests(harness) {
+  harness
+    .runFrontendSession([
+      testResume,
+    ]).expectShutDown(0);
+}
+
+const script = 'process._debugEnd();' +
```

Comment:
_debugProcess is useful, frontends use it to attach to running instances.

__debugEnd, on the other hand, can only be called on self so the application needs to explicitly "want" to make itself undebuggable. I am not sure why...